### DISCUSSION TITLE
Move the Muse Hub

### DISCUSSION THREAD

**Discussion by pssss4ssss** [musescore/MuseScore#15393] [Ideas]
I wish I could move the Muse Hub around in my desktop, the right bottom part isn't the best place to put it for me, it doesn't really look pretty, it would probably be cool if it were a window maybe.

**Comment by jeetee** [comment]
See https://musehub.zendesk.com/hc/en-gb/community/posts/8436619631389-I-want-to-move-muse-hub-window- where it seems there is currently no intention of making the application an actual application..
